src/support.h

```diff
@@ -4,6 +4,9 @@
 #ifndef SUPPORT_H
 #define SUPPORT_H
 
+#include <memory>
+#include "BeadingStrategy/BeadingStrategy.h"
```

Comment:
Move this include to the .cpp file, where it's actually being used.

Reply:
Remnant of an old test, will move it to the .cpp